CMakeLists.txt

```diff
@@ -12,14 +12,8 @@
 #
 set(ITK_OLDEST_VALIDATED_POLICIES_VERSION "3.16.3")
 set(ITK_NEWEST_VALIDATED_POLICIES_VERSION "3.19.7")
-cmake_minimum_required(VERSION ${ITK_OLDEST_VALIDATED_POLICIES_VERSION} FATAL_ERROR)
-if("${CMAKE_VERSION}" VERSION_LESS_EQUAL "${ITK_NEWEST_VALIDATED_POLICIES_VERSION}")
-  #Set and use the newest available cmake policies that are validated to work
-  set(ITK_CMAKE_POLICY_VERSION "${CMAKE_VERSION}")
-else()
-  set(ITK_CMAKE_POLICY_VERSION "${ITK_NEWEST_VALIDATED_POLICIES_VERSION}")
-endif()
-cmake_policy(VERSION ${ITK_CMAKE_POLICY_VERSION})
+cmake_minimum_required(VERSION ${ITK_OLDEST_VALIDATED_POLICIES_VERSION}..${ITK_NEWEST_VALIDATED_POLICIES_VERSION} FATAL_ERROR)
```

Comment:
`cmake_minimum_required(VERSION <min>[...<policy_max>] [FATAL_ERROR])`

Looks like 3 dots?

Reply:
Fixed.  Thanks for the close inspection.